docs/linter-rules/adminUsername-should-not-be-literal.md

```diff
@@ -0,0 +1,49 @@
+# AdminUserName should not be a literal
```

Comment:
@MarcusFelling Please review

Reply:
@mumian will we need to sync with you when adding new linter rules so you can add them to the new docs location?

Reply:
A new aka link will need to be added and pointed at docs, this case, https://docs.microsoft.com/en-us/azure/azure-resource-manager/bicep/adminUsername-should-not-be-literal